lib/OpenQA/WebAPI/Plugin/ObsRsync/Controller/Gru.pm

```diff
@@ -128,7 +128,7 @@ sub run {
     return $self->render(json => {message => 'queue full'}, status => QUEUE_FULL)
       if ($results->{total} >= $queue_limit);
 
-    $app->gru->enqueue('obs_rsync_run', {project => $project}, {priority => 100});
+    $app->gru->enqueue('obs_rsync_run', {project => $project, url => $helper->project_status_url}, {priority => 100});
```

Comment:
I can't make sense of the unit test failure. Adding the `url` argument somehow causes the list of inactive projects to be different. Changing `_extend_job_info` doesn't seem relevant. There must be something else?

Reply:
The adjustment needs to be done here as well: https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/WebAPI/Plugin/ObsRsync/Controller/Folders.pm#L75
```
            next unless (ref $args eq 'HASH' && scalar(%$args) == 1 && $args->{project});
```

Reply:
That was it! Thank you so much!